Rakefile

```diff
@@ -17,7 +17,7 @@ end
 begin
   require 'rspec/core/rake_task'
   RSpec::Core::RakeTask.new(:spec_with_profile) do |t|
-    t.rspec_opts = '--profile'
+    t.rspec_opts = '--profile --exclude-pattern "spec/performance/**/*.rb"'
```

Comment:
Hmm, we're not running `spec_with_profile` anymore after https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/807, is there another place this could go for default?

Reply:
I've updated this after merging in your PR #807